From 63e157666c12a834bc1910d7c8727ad95e9e76b9 Mon Sep 17 00:00:00 2001
From: Simon Marchi <simon.marchi@polymtl.ca>
Date: Mon, 10 Aug 2015 15:01:42 -0400
Subject: [PATCH] Remove -fvisibiliy-inlines-hidden

Signed-off-by: Simon Marchi <simon.marchi@polymtl.ca>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 819835218..55f818ca1 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -45,7 +45,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --enable-python-libs
 COMMON_CXXFLAGS_ONLY_WARNINGS = \
     -I$(top_srcdir)/include \
     -I$(top_builddir)/include \
-    -Wall -Wformat -W -fvisibility-inlines-hidden \
+    -Wall -Wformat -W \
     $(libprotobuf_CFLAGS)
 
 COMMON_CXXFLAGS = $(COMMON_CXXFLAGS_ONLY_WARNINGS)
-- 
2.14.3

